src/composite/put_block.jl

```diff
@@ -104,3 +104,60 @@ function YaoBase.iscommute(x::PutBlock{N}, y::PutBlock{N}) where N
         return iscommute_fallback(x, y)
     end
 end
+
+const Swap{N} = PutBlock{N,2,G} where G<:ConstGate.SWAPGate
```

Comment:
can you change the printings as well?

Reply:
The printing is ok, you mean the printing for PSwap?